addons/web_unsplash/controllers/main.py

```diff
@@ -115,7 +113,14 @@ def slugify(s):
                 'res_id': res_id,
                 'res_model': res_model,
                 'description': value.get('description'),
-            })
+            }
+            if not request.env.user._has_unsplash_key_rights():
```

Comment:
that condition is weird, it will apply the portal logic to all non-admin users, not just portal?

In order to simplify and unify the logic, could we call web_editor's `_attachment_create()` here, wrapped with the unsplash-specific stuff?